### DISCUSSION TITLE
what is affect on battery life when all watchers are enabled.

### DISCUSSION THREAD

**Discussion by opensusefan** [ActivityWatch/activitywatch#906] [General]
I just stumbled upon activity watcher so don't know how it works; does it runs its watchers in background all the time?

**Comment by BelKed** [comment]
Yes, it always runs its watchers in the background unless you don't quit them.
And, I've not noticed any significant drop in battery life or something while using ActivityWatch.

**Reply by opensusefan** [reply]
thanks for this info
